Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -155,15 +155,15 @@ open class AVFoundationPlayback: Playback {
 
     open override var position: Double {
         if isDvrAvailable, let start = dvrWindowStart,
```

Comment:
```suggestion
        if let start = dvrWindowStart,
```
We don't need to check that here, cause it's checked inside the `dvrWindowStart` property.